src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -546,8 +546,7 @@ void SotaUptaneClient::getNewTargets(std::vector<Uptane::Target> *new_targets, u
       // TODO(OTA-4939): Unify this with the check in
       // PackageManagerFake::fetchTarget() and make it more generic.
       if (primary_ecu_serial == ecu_serial) {
-        if (!target.IsOstree() &&
-            (config.pacman.type == PACKAGE_MANAGER_OSTREE || config.pacman.type == PACKAGE_MANAGER_OSTREEDOCKERAPP)) {
+        if (!target.IsOstree() && config.pacman.type == PACKAGE_MANAGER_OSTREE) {
```

Comment:
How do you deal with this in aktualizr-lite? Obviously we have the TODO/ticket to improve this, but we also have some (internal/private) code that has to work around this type of issue.

Reply:
We don't go through that code path in our code base.